pyccel/codegen/compiling/compilers.py

```diff
@@ -28,7 +28,7 @@
     mac_target = '{}.{}'.format(*mac_version_tuple[:2])
     os.environ['MACOSX_DEPLOYMENT_TARGET'] = mac_target
 
-def get_condaless_search_path():
+def get_condaless_search_path(is_conda_warnings_disabled = False, is_conda_warnings_detailed = False):
     """ Get the value of the PATH variable to be set when searching for the compiler.
     This is the same as the environment PATH variable but without any conda paths
     """
```

Comment:
I'm not sure how this slipped through the docstring tests, but this does not have the right format. Would you mind fixing it

Reply:
This was missed due to a bug which is fixed in #1406 . I will try to get that PR merged ASAP but if it takes too long you may need to merge it here manually

Reply:
The PR is now merged. There are now several new errors. But most of them are just about the formatting

Reply:
https://github.com/pyccel/pyccel/blob/e1f3ebeea7a4b621de7f03408aa23eda26a87764/pyccel/codegen/compiling/compilers.py#L32-L50